### ISSUE TITLE
Sandboxed workers... starting infinitely?

### ISSUE THREAD

**Issue by paulsingh** [taskforcesh/bullmq#1533]
I've been using BullMQ 2 without any problems for a few months now. Earlier today, I updated to BullMQ 3.2 and quickly started running out of memory on my worker machines. After digging around a bit, I noticed that the output of `ps -ef` shows hundreds of these lines: 

/usr/bin/node /home/******/*****************/releases/20221110041353516/node_modules/bullmq/dist/cjs/classes/master.js

Digging around the Issues section, it seems that this _might_ be related to the fact that I'm using sandboxed workers. 🤷🏻‍♂️

I'm happy to provide more information but, candidly, I'm still getting my bearings with the problem itself. Any chance this is something others are experiencing recently?

**Comment by manast** [comment]
There are no changes related to sandboxed workers between v2 and v3, maybe it is a coincidence and the error was in v2 too but you did not noticed it until now?

**Comment by paulsingh** [comment]
Thank you for the response. As I was looking at the various commits on this repo, I came to the conclusion that this must be something in my own code... 

Turns out, I had added a new worker yesterday and didn't properly handle errors within that specific sandboxed worker. For whatever reason, that caused thousands of child process to pile up. Once I commented out yesterday's new worker, everything went back to normal.

Closing this issue for now -- apologies for the scramble!
